ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -601,55 +597,66 @@ objects:
+            cp "${OS_UPI_DIR}/down-03_bootstrap.yaml"       "${ASSETS_DIR}/down-bootstrap.yaml"
+            cp "${OS_UPI_DIR}/down-04_control-plane.yaml"   "${ASSETS_DIR}/down-control-plane.yaml"
+            cp "${OS_UPI_DIR}/down-05_compute-nodes.yaml"   "${ASSETS_DIR}/down-compute-nodes.yaml"
+            cp "${OS_UPI_DIR}/down-06_load-balancers.yaml"  "${ASSETS_DIR}/down-load-balancers.yaml"
+          else
+            cp \
+              "${OS_UPI_DIR}/security-groups.yaml"      \
+              "${OS_UPI_DIR}/network.yaml"              \
+              "${OS_UPI_DIR}/bootstrap.yaml"            \
+              "${OS_UPI_DIR}/control-plane.yaml"        \
+              "${OS_UPI_DIR}/compute-nodes.yaml"        \
+                                                        \
+              "${OS_UPI_DIR}/down-security-groups.yaml" \
+              "${OS_UPI_DIR}/down-network.yaml"         \
+              "${OS_UPI_DIR}/down-bootstrap.yaml"       \
+              "${OS_UPI_DIR}/down-control-plane.yaml"   \
+              "${OS_UPI_DIR}/down-compute-nodes.yaml"   \
+              "${OS_UPI_DIR}/down-load-balancers.yaml"  \
+                                                        \
+              "${ASSETS_DIR}"
           fi
-        }
 
-        function prepare_ansible_playbooks_workspace() {
           OPENSTACK_REGION="$(get_clouds_param 'region_name')"
-          echo "OpenStack UPI directory: ${OS_UPI_DIR}"
-          #prepare inventory.yaml
-          cp ${OS_UPI_DIR}/inventory.yaml ${OS_UPI_DIR}/inventory.yaml.bkk
-          sed -i "s~os_subnet_range.*~os_subnet_range: \'${OS_SUBNET_RANGE}\'~g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_flavor_master.*/os_flavor_master: \'${OPENSTACK_FLAVOR}\'/g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_flavor_worker.*/os_flavor_worker: \'${OPENSTACK_FLAVOR}\'/g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_image_rhcos.*/os_image_rhcos: \'${RHCOS_GLANCE_IMAGE_NAME}\'/g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_external_network.*/os_external_network: \'${OPENSTACK_EXTERNAL_NETWORK}\'/g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_api_fip.*/os_api_fip: \'${LB_FIP_IP}\'/g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_ingress_fip.*/os_ingress_fip: \'${INGRESS_FIP_IP}\'/g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_region_name.*/os_region_name: \'${OPENSTACK_REGION}\'/g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_cp_nodes_number.*/os_cp_nodes_number: ${NUMBER_OF_MASTERS}/g" ${OS_UPI_DIR}/inventory.yaml
-          sed -i "s/os_compute_nodes_number.*/os_compute_nodes_number: ${NUMBER_OF_WORKERS}/g" ${OS_UPI_DIR}/inventory.yaml
-
-          # The metadata.json  and ignition json files needs to be in the same directory as playbooks
-          cp ${ASSETS_DIR}/*.json ${OS_UPI_DIR}/
-          cp ${ASSETS_DIR}/*.ign ${OS_UPI_DIR}/
+          sed "
+            s~os_subnet_range.*~os_subnet_range: \'${OS_SUBNET_RANGE}\'~;
```

Comment:
any reason to use a different marker `~`? Might want to use `/` to be consistent with below lines.

Reply:
`$OS_SUBNET_RANGE` will be substituted with an IP range in the form `192.168.0.0/16`, so we have to use a different separator or sed will panic